### ISSUE TITLE
SMS Conversation window allows other users into the conference.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#48]
```
What steps will reproduce the problem?
1. Set the options to have individual chat window per SMS conversation.
2. Opens up a chat conference.
3. Other people are somehow able to enter this chant conversation.

What is the expected output? What do you see instead?
Only allow you and the SMS recipient to occupy the room. I see other people 
entering the conversation window and they were able to chat to me and 
potentially my SMS contact. 

What version of the product are you using? On what operating system?
1.7.1


```

Original issue reported on code.google.com by `i.vita...@gmail.com` on 13 Jan 2011 at 3:56

**Comment by GoogleCodeExporter** [comment]
```
It's a private room, so they can connect only if they know the room name.
The name is gtalksms_[rand number]_[login]@conference.jwchat.org

Do you give the room name for test purpose or somebody has find your private 
room ?

For now I've some problems to set room owners (not implemented by server?) that 
why I don't set this feature by default

```

Original comment by `Florent....@gmail.com` on 13 Jan 2011 at 4:05

**Comment by GoogleCodeExporter** [comment]
```
I just tried the feature. And then someone randomly logged in. I don't know 
how, but then they sent me a private message asking how I was. 
```

Original comment by `i.vita...@gmail.com` on 13 Jan 2011 at 4:15

**Comment by GoogleCodeExporter** [comment]
```
Maybe this person has the same login as you but on another domain, per example 
bob@gmail.com & bob@jabber.org

I'll investigate
```

Original comment by `Florent....@gmail.com` on 13 Jan 2011 at 4:58

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 14 Jan 2011 at 1:54
- Changed state: **Investigating**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
I've added password protection. That should solve your problem.
Fixed in next release
```

Original comment by `Florent....@gmail.com` on 16 Jan 2011 at 5:22
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.7.2
```

Original comment by `Florent....@gmail.com` on 16 Jan 2011 at 7:43
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
